pkg/auth/iam/event.go

```diff
@@ -44,7 +44,7 @@ const (
 	InvalidPaginationParameters = 20021
 	TokenIsNotUserToken         = 20022
 	InvalidRefererHeader        = 20023
-	SubdomainMismatch           = 20024
+	SubdomainMismatch           = 20027
```

Comment:
unfortunately it has been used: https://bitbucket.org/accelbyte/justice-common-core/src/master/src/main/java/net/accelbyte/platform/util/ErrorCodes.java

Reply:
if not occupied, you can use 20030

Reply:
Thanks for the information. Yep, I'll use 20030